pandas/core/groupby/groupby.py

```diff
@@ -1131,10 +1131,16 @@ def _transform_with_numba(self, data, func, *args, engine_kwargs=None, **kwargs)
         group_keys = self.grouper._get_group_keys()
 
         numba_transform_func = numba_.generate_numba_transform_func(
-            tuple(args), kwargs, func, engine_kwargs
+            kwargs, func, engine_kwargs
```

Comment:
shouldn't knwargs be handled in the same way?

Reply:
numba doesn't support `**kwargs` in udf's currently. It's passed here to raise an error if kwargs is not empty